src/tests.rs

```diff
@@ -86,18 +142,16 @@ pub mod serverlesstest {
                     running: AtomicBool::new(true),
                     runtime_path: "./runtime/".to_owned(),
                 }))
-                .configure(handler::config),
+                .default_service(web::to(handler::serverless)),
```

Comment:
This test for invalid TxHash is technically the same as interacting with wrong smart contract, since there is no verification of the string of the TxHash.